keyboards/handwired/brain/config.h

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+#define PRODUCT     Moduform
+#define DIODE_DIRECTION COL2ROW
+
+#define SPLIT_USB_DETECT
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 7
+
+// wiring of each half
+
+#define MATRIX_COL_PINS { D1, D4, C6, D7, E6, B4, B5 }
+#define MATRIX_ROW_PINS { F4, F5, F6, F7, B1 }
+
+#define MATRIX_COL_PINS_RIGHT { B5, B4, E6, D7, C6, D4, D1 }
+
+
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x3060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    klackygears
+// defined in subfolder
+#define DESCRIPTION     A split ergonomic keyboard with sculpted shell
+
+/* mouse config */
+#define MOUSEKEY_INTERVAL       20
+#define MOUSEKEY_DELAY          0
+#define MOUSEKEY_TIME_TO_MAX    60
+#define MOUSEKEY_MAX_SPEED      7
+#define MOUSEKEY_WHEEL_DELAY 0
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 0
```

Comment:
Would be simpler to undefine, actually.

```suggestion
```

Reply:
This file was pretty messy! Thanks for looking it over!